### PR TITLE
[LAYOUT] Footer - OT102-22

### PR DESCRIPTION
**Pull request by Gabriiel99**
Description
HOW: User
I WANT: View the footer of the page
TO: Have extra information about it

Acceptance criteria: Must show name, logo and link to social networks of the Organization. This data will be obtained from the public data endpoint.

3 columns

1.Logo of the ONG With name

2.Web links

3.Social media links
Captura

### CODE REVIEW

**Review by JonatanGarbuyo — APPROVED** [review]
parece que todo esta ok.  pero fallaron los checks..

**Review by dvelezroman — APPROVED** [review]
se ve bien.. solo arreglar ese conflicto que hay cunado buildea.

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+/* eslint-disable import/no-unresolved */
 import React from 'react'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
 
+// eslint-disable-next-line import/extensions
+import Footer from './components/header/Footer'
```
Comment (dvelezroman) [inline src/App.js]: cambiar ruta

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
+/* eslint-disable import/no-unresolved */
 import React from 'react'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
 
+// eslint-disable-next-line import/extensions
```
Comment (dvelezroman) [inline src/App.js]: quitar este eslint error

**Review by dvelezroman — CHANGES_REQUESTED** [review]
pequenios cmabios

**Review by dvelezroman — APPROVED** [review]

**Review by gabyMangoni — APPROVED** [review]

**Pull request merged**